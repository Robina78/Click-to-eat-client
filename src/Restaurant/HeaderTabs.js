import React from 'react';
import { Container } from "react-bootstrap";
import "./HeaderTabs.css";

export default function HeaderTabs(props) {    

    return (
        <Container>
           <HeaderButton
            text="Delivery"
            btnColor="slateblue"
            textColor="white"            
            activeTab={props.activeTab}
            setActiveTab={props.setActiveTab}
           />
           <HeaderButton
            text="Pickup"
            btnColor="white"
            textColor="slateblue"
            activeTab={props.activeTab}
            setActiveTab={props.setActiveTab}
           />  
        </Container>
    )
}

const HeaderButton = (props) => {
    return (
        <div>
      <button className="btn" style={{
        backgroundColor: props.activeTab === props.text ? "slateblue" : "white",
        width:120,
        height:40,       
        borderRadius: 30,
        color: props.activeTab === props.text ? "white": "slateblue",
        fontSize: 16,
        fontWeight: "900",
        marginBottom:'20'       
        }}
        onClick={() => props.setActiveTab(props.text)}
      >{props.text}
      </button>
    </div>
    )
    
}
