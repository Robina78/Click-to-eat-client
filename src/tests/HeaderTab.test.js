import React from "react";
import { render } from "@testing-library/react";
import HeaderTabs from "../Restaurant/HeaderTabs";


it("matches snapshot", function () {
  const { asFragment } = render(<HeaderTabs />);
  expect(asFragment()).toMatchSnapshot();
});