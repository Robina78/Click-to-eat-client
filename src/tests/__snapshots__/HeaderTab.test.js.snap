// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div>
      <button
        class="btn"
        style="background-color: white; width: 120px; height: 40px; border-radius: 30px; color: slateblue; font-size: 16px; font-weight: 900;"
      >
        Delivery
      </button>
    </div>
    <div>
      <button
        class="btn"
        style="background-color: white; width: 120px; height: 40px; border-radius: 30px; color: slateblue; font-size: 16px; font-weight: 900;"
      >
        Pickup
      </button>
    </div>
  </div>
</DocumentFragment>
`;
